References
============

Credits & Acknowledgements
+++++++++++++++++++++++++++

The pipeline is based on `nf-core's <https://pubmed.ncbi.nlm.nih.gov/32055031/>`_ `AmpliSeq <https://nf-co.re/ampliseq/2.1.0>`_ pipeline version 2.1.0. 

**nf-core**
    Ewels, P., Peltzer, A., Fillinger, S., Patel, H., Alneberg, J., Wilm, A., Garcia, M. U., Di Tommaso, P., & Nahnsen, S. (2020). The nf-core framework for community-curated bioinformatics pipelines. Nature biotechnology, 38(3), 276–278. `https://doi.org/10.1038/s41587-020-0439-x`

**AmpliSeq release 2.1.0**
    Straub, D., Peltzer, A., Lundin, D., Tångrot, J., Emnilsson, DiegoBrambilla, Bot, N.-C., Gabernet, G., PhilPalmer, Garcia, M. U., Davenport, C., Ewels, P., Patel, H., & Menden, K. (2021, September 14). NF-core/ampliseq: Ampliseq version 2.1.0 “gray steel boa”. `https://doi.org/10.5281/zenodo.5506988`

**AmpliSeq introduction paper**
    Straub, D., Blackwell, N., Langarica-Fuentes, A., Peltzer, A., Nahnsen, S., & Kleindienst, S. (2020). Interpretations of Environmental Microbial Community Studies Are Biased by the Selected 16S rRNA (Gene) Amplicon Sequencing Pipeline. Frontiers in microbiology, 11. `https://doi.org/10.3389/fmicb.2020.550420` PMID: 33193131; PMCID: PMC7645116.

Tools & Assets
+++++++++++++++++

**Nextflow**
    Di Tommaso, P., Chatzou, M., Floden, E. W., Barja, P. P., Palumbo, E., & Notredame, C. (2017). Nextflow enables reproducible computational workflows. Nature biotechnology, 35(4), 316–319. `https://doi.org/10.1038/nbt.3820`. PubMed PMID: 28398311.

**FastQC**
    Andrews, S. (2019). A Quality Control tool for High Throughput Sequence Data. Babraham Bioinformatics. `https://www.bioinformatics.babraham.ac.uk/projects/fastqc/`

**Cutadapt**
    Martin, M. (2011). Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet.journal, 17(1). `https://doi.org/10.14806/ej.17.1.200`

**DADA2**
    Callahan, B. J., McMurdie, P. J., Rosen, M. J., Han, A. W., Johnson, A. J. A., & Holmes, S. P. (2016). DADA2: High-resolution sample inference from Illumina amplicon data. Nature Methods, 13(7). `https://doi.org/10.1038/nmeth.3869`

**SILVA database**
    Quast, C., Pruesse, E., Yilmaz, P., Gerken, J., Schweer, T., Yarza, P., Peplies, J., & Glöckner, F. O. (2013). The SILVA ribosomal RNA gene database project: improved data processing and web-based tools. Nucleic Acids Research, 41(D1). `https://doi.org/10.1093/nar/gks1219`

**QIIME2**
    Bolyen, E., Rideout, J. R., Dillon, M. R., Bokulich, N. A., Abnet, C. C., Al-Ghalith, G. A., Alexander, H., Alm, E.J., Arumugam, M., Asnicar, F., Bai, Y., Bisanz, J. E., Bittinger, K., Brejnrod, A., Brislawn, C. J., Brown, C. T., Callahan, B. J., Caraballo-Rodríguez, A. M., Chase, J., Cope, E. K., Da Silva, R., Diener, C., Dorrestein, P. C., Douglas, G. M., Durall, D. M., Duvallet, C., Edwardson, C. F., Ernst, M. , Estaki, M., Fouquier, J., Gauglitz, J. M., Gibbons, S. M., Gibson, D. L., Gonzalez, A., Gorlick, K., Guo, J., Hillmann, B., Holmes, S., Holste, H., Huttenhower, C., Huttley, G. A., Janssen, S., Jarmusch, A. K., Jiang, L., Kaehler, B. D., Kang, K. B., Keefe, C. R., Keim, P., Kelley, S.T., Knights, D., Koester, I., Kosciolek, T., Kreps, J., Langille, M. G. I., Lee, J., Ley, R., Liu, Y. X., Loftfield, E., Lozupone, C., Maher, M., Marotz, C., Martin, B. D., McDonald, D., McIver, L. J., Melnik, A. V., Metcalf, J. L., Morgan, S. C., Morton, J. T., Naimey, A. T., Navas-Molina, J. A., Nothias, L. F., Orchanian, S. B., Pearson, T., Peoples, S. L., Petras, D., Preuss, M. L., Pruesse, E., Rasmussen, L. B., Rivers, A., Robeson, M. S., Rosenthal, P., Segata, N., Shaffer, M., Shiffer, A., Sinha, R., Song, S. J., Spear, J. R., Swafford, A. D., Thompson, L. R., Torres, P. J., Trinh, P., Tripathi, A., Turnbaugh, P. J., Ul-Hasan, S., van der Hooft, J. J. J., Vargas, F., Vázquez-Baeza, Y., Vogtmann, E., von Hippel, M., Walters, W., Wan, Y., Wang, M., Warren, J., Weber, K. C., Williamson, C. H. D, Willis A. D., Xu, Z. Z., Zaneveld, J. R., Zhang, Y., Zhu, Q., Knight, R., & Caporaso, J. G. (2019). Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2. Nature Biotechnology, 37(8). `https://doi.org/10.1038/s41587-019-0209-9`

**MultiQC**
    Ewels, P., & Magnusson, M. (2016). MultiQC: summarize analysis results for multiple tools and samples in a single report. Oxford Academic Bioinformatics, 32(19). `https://doi.org/10.1093/bioinformatics/btw354`

**KronaTools**
    Ondov, B. D., Bergman, N. H., & Phillippy, A. M. (2011). Interactive metagenomic visualization in a Web browser. BMC bioinformatics, 12, 385. `https://doi.org/10.1186/1471-2105-12-385`

